docs/code_contribution_guidelines.md

```diff
@@ -572,7 +572,7 @@ Rejoice as you will now be listed as a [contributor](https://github.com/lightnin
 
 ## Contribution Checklist
 
-- [&nbsp;&nbsp;] All changes are Go version 1.12 compliant
+- [&nbsp;&nbsp;] All changes are Go version 1.13 compliant
```

Comment:
This should actually be Go 1.14 since we usually support the two latest versions rn: 1.14 and 1.15. However Go 1.16 has also been released, so perhaps we make this Go 1.15 instead

Reply:
Updated